frontend/src/metabase/containers/DataPicker/tests/common.tsx

```diff
@@ -0,0 +1,178 @@
+import React from "react";
```

Comment:
Will this file get included in test coverage reports since its not a `*.spec.*` file? maybe it should have a `/* istanbul ignore file */` at the top?